fe/fe-core/src/main/java/com/starrocks/sql/parser/AstBuilder.java

```diff
@@ -2208,16 +2208,26 @@ public ParseNode visitShowComputeNodesStatement(StarRocksParser.ShowComputeNodes
 
     // ------------------------------------------- Analyze Statement ---------------------------------------------------
 
+    private List<Expr> getAnalyzeColumns(List<QualifiedName> qualifiedNames) {
+        List<Expr> columns = Lists.newArrayList();
+        for (QualifiedName qualifiedName : qualifiedNames) {
```

Comment:
If user writes clause like `tableName.intColName`, it seems not be processed correctly.

Reply:
If use writes clause like tableName.ColName, it will throw exception at analyze stmt analyzer, and report it can not find the column